sdk/test-utils/recorder/test/node/utils.spec.ts

```diff
@@ -394,4 +395,15 @@ describe("NodeJS utils", () => {
       });
     });
   });
+
+  describe("fs helpers", () => {
+    it("should be able to create a folder if doesn't exists", function() {
+      createFolderForRecording("./tmp/temp/temporary");
+      createFolderForRecording("./tmp/temp/temporary2/");
```

Comment:
Is the path that is passed in always a relative path starting with `.`? If an absolute path is passed, am not sure if the current logic in `createFolderForRecording` would work